docs/devices/4523430.md

```diff
@@ -72,7 +72,7 @@ The unit of this value is `min`.
 ### Climate 
 This climate device supports the following features: `occupied_heating_setpoint`, `local_temperature`, `local_temperature_calibration`, `system_mode`, `running_state`.
 - `occupied_heating_setpoint`: Temperature setpoint. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"occupied_heating_setpoint": VALUE}` where `VALUE` is the °C between `5` and `50`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"occupied_heating_setpoint": ""}`.
-- `local_temperature`: Current temperature measured on the device (in °C). Reading (`/get`) this attribute is not possible.
+- `local_temperature`: Current temperature measured on the device (in °C). Reading (`/get`) It's not possible to write (`/set`) this value. 
```

Comment:
This line is generated from https://github.com/Koenkk/zigbee2mqtt.io/blob/2e10ab236c0ae9d2573954c49f8508600951c431/docgen/device_page_exposes.ts#L266, could you update that instead?